base/show.jl

```diff
@@ -1031,7 +1031,7 @@ function show_unquoted_quote_expr(io::IO, @nospecialize(value), indent::Int, pre
             print(io, ":")
             print(io, value)
         else
-            print(io, "Symbol(\"", escape_string(s), "\")")
+            print(io, "Symbol(\"", escape_string(s, "\"\$"), "\")")
```

Comment:
replace `print(io, "Symbol(\"", escape_string(s, "\"\$"), "\")")` with `print(io, "Symbol(", repr(s), ")")`?

Reply:
ok

Reply:
might even be good to go through and do a total cleanup pass of the code, since there aren't many left